loader/db.go

```diff
@@ -16,11 +16,14 @@ package loader
 import (
 	"database/sql"
 	"fmt"
-	"time"
-
 	"github.com/pingcap/dm/dm/config"
 	tcontext "github.com/pingcap/dm/pkg/context"
 	"github.com/pingcap/dm/pkg/log"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
```

Comment:
we split import to three part:
1. the standard pkg, include these "time", "strings" and so on
2. pkg in dm, like the "github.com/pingcap/dm/dm/config"
3. and other third party pkg

please format this import

Reply:
"regexp"，"strconv" and "strings" exists in the previous version，I just add a "regexp"

Reply:
I means we can move these five to the top, together with "fmt"